charts/incubator/proxmox-backup-server/values.yaml

```diff
@@ -0,0 +1,41 @@
+image:
+  repository: tccr.io/truecharts/proxmox-backup-server
+  tag: v2.2-7@sha256:dcbaee6378f32c5062518ea39a5e8e1260c3a3c609bcb32bd2dd406b7a9ef0d0
+  pullPolicy: IfNotPresent
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsNonRoot: false
+    runAsUser: 0
+    runAsGroup: 0
+    fsGroup: 43
+
+service:
+  main:
+    ports:
+      main:
+        targetPort: 8007
+        protocol: https
+        port: 8007
+
+persistence:
+  config:
+    enabled: true
+    mountPath: /etc/proxmox-backup
+  lib:
+    enabled: true
+    type: emptyDir
+    mountPath: /var/lib/proxmox-backup
```

Comment:
This will mount an **empty** dir in this location (wiping any existing content)
Is that desired?

Reply:
yes